.github/workflows/benefits-finder-test.yml

```diff
@@ -0,0 +1,173 @@
+name: Benefits Finder Integration Test
+
+on:
+  workflow_dispatch:
```

Comment:
After confirmation that the dispatch works as we expect, consider updating this to trigger on PR's that had changes in the `/web/themes` directory